### PR TITLE
Update Release documentation for 252 and 261 and add explanatory blog post

### PR DESCRIPTION
**Pull request by aindriu-aiven**
Update Release documentation for 252 and 261 and add explanatory blog post

### CODE REVIEW

File: blog/2023/11/important-community-update-klaws-latest-actions-to-enhance-transparency.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    kafka,
+    governance,
+    klaw,
+    schema registry,
+    disaster recovery,
+    migration,
+    evolution,
+  ]
+date: Sep 5, 2023
+---
```
Comment (harshini-rangaswamy) [inline blog/2023/11/important-community-update-klaws-latest-actions-to-enhance-transparency.md:16]: I think this slug info can be removed as this is not the current blog.

File: blog/2023/11/important-community-update-klaws-latest-actions-to-enhance-transparency.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+slug: schema-management-disaster-recovery-in-klaw
```
Comment (muralibasani) [inline blog/2023/11/important-community-update-klaws-latest-actions-to-enhance-transparency.md:2]: I think we need to update these .. slug, title, author, tags

**Review by muralibasani — APPROVED** [review]
LGTM, thanks

File: blog/2023/11/important-community-update-klaws-latest-actions-to-enhance-transparency.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Scope and deactivation of the tracker:
+This tracking code was specifically configured to gather data from kafkawize.com. We want to reassure our community that this means that no data from any other host has been or is being collected, accepted or stored by this code. As of February 2022, this configuration has been completely removed.
+
+Impact on users:
+We are pleased to inform you that the legacy code has not affected our newer Coral React UI. This interface, representing the latest development of our platform, remains free from any impact of this outdated tracking code.
+
+In commitment to our community's transparency, we are releasing software updates 2.5.2 and 2.6.1. These versions aim to completely remove this legacy code, allowing our users to transition to a version where their privacy and trust are further safeguarded.
+
+Your feedback is important. Stay updated and reach out to us via [GitHub/Email] for any queries or suggestions.
```
Comment (harshini-rangaswamy) [inline blog/2023/11/important-community-update-klaws-latest-actions-to-enhance-transparency.md:26]: ``via [GitHub/Email]``

I just noticed this. Can we update this with a link to the GH repo?

Comment (aindriu-aiven) [reply blog/2023/11/important-community-update-klaws-latest-actions-to-enhance-transparency.md:26]: yes

**Review by muralibasani — DISMISSED** [review]

**Review by harshini-rangaswamy — APPROVED** [review]
Look good, thanks!

**Pull request merged**